src/sentry/event_manager.py

```diff
@@ -2266,7 +2268,8 @@ def _calculate_event_grouping(
             event.data["grouping_config"] = get_grouping_config_dict_for_project(project)
             hashes = event.get_hashes()
 
-    hashes.write_to_event(event.data)
+    # Using cast to satisfy mypy
+    hashes.write_to_event(cast(Dict[str, Any], event.data))
```

Comment:
I was not sure how to fix this one without getting complicated.
I was getting this message before: `src/sentry/event_manager.py:2269: error: Argument 1 to "write_to_event" of "CalculatedHashes" has incompatible type "NodeData"; expected "Dict[str, Any]"  [arg-type]`

Here's the code for [write_to_event](https://github.com/getsentry/sentry/blob/db9df6f3a513c9ebc821d0382e2b7e78637696bb/src/sentry/grouping/result.py#L100)

Reply:
`cast` should only be used for mypy bugs -- you'll need to fix the types here or convert -- probably by fixing the signature for `write_to_event`